src/components/Facility/FacilityUsers.tsx

```diff
@@ -86,7 +86,7 @@ export default function FacilityUsers(props: { facilityId: string }) {
           onChange={(e) => updateQuery({ username: e.target.value })}
           value={qParams.username}
           placeholder={t("search_by_username")}
-          className="w-full max-w-sm"
+          className="w-full max-w-sm p px-2 py-1 placeholder:text-xs max-[350px]:placeholder:text-[8px] "
```

Comment:
1. there are invalid classes here.
2. we do not use such a small font-size in care, let's not deviate away from design system. we could move the search bar down.
3. discard changes to lockfile